### PR TITLE
update for suppression config

### PR DESCRIPTION
**Pull request by yoid2000**
The image still needs to be updated (I'll do after I have a version of the app to play with).

I'm leaving changing the location of the suppression threshold to a separate issue/pull request.

### CODE REVIEW

File: docs/operation.md [hunk 1]
```diff
@@ -2,8 +2,11 @@
 
 > To report feature requests or problem, please contact us at [feedback@open-diffix.org](mailto:feedback@open-diffix.org).
 
-__Diffix for Desktop__ has three phases of operation:
-- Load and configure table from CSV
+__Diffix for Desktop__ has four phases of operation:
+- Load table from CSV
+- Configuration anonymization parameters
```
Comment (edongashi) [inline docs/operation.md:7]: **Configure** anonymization parameters?

File: docs/operation.md [hunk 2]
```diff
@@ -16,13 +19,11 @@ An unlimited number of anonymized views of the data may be exported without comp
...
-__Diffix for Desktop__ interprets the first row of the CSV file as column names.
-
-__Diffix for Desktop__ auto-detects the CSV separator. All standard separators are accepted.
-
-__Diffix for Desktop__ auto-detects data types as text or numeric. Text columns are generalized with substring selection, and numeric columns are generalized with numeric ranges.
+__Diffix for Desktop:__ 
+- Only accepts CSV files as input.
+- Interprets the first row of the CSV file as column names.
+- Auto-detects the CSV separator. All standard separators are accepted.
+- Auto-detects data types as text or numeric. Text columns are generalized with substring selection, and numeric columns are generalized with numeric ranges.
```
Comment (edongashi) [inline docs/operation.md:26]: We also have boolean detection.

File: docs/operation.md [hunk 3]
```diff
@@ -77,6 +78,86 @@ The `Sender email` and `Receiver email` each identify a different protected enti
...
 A data set with multiple protected entities needs to be pre-processed to have one protected entity per row before loading into __Diffix for Desktop__. See [Multiple protected entities per row](tips.md#multiple-protected-entities-per-row).
 
+## Suppression threshold configuration
+
+__Diffix for Desktop__ suppresses bins that are comprised of too few protected entities.
+This hides individual data values or combinations of data values that could
+leak private information about individual protected entities.
+
+Different output bins have different suppression thresholds: the threshold used for any
+given output bin is a random value taken from a Normal distribution. There is, however, a hard
```
Comment (edongashi) [inline docs/operation.md:88]: No reason to capitalize **Normal** distribution

File: docs/operation.md [hunk 4]
```diff
@@ -77,6 +78,86 @@ The `Sender email` and `Receiver email` each identify a different protected enti
...
+to generate output bins comprised of more protected entities is via column generalization.
+
+There may also be a *data quality* benefit to setting `suppression threshold` to a higher
+value. This is because bins with very few protected entities have more relative distortion
+than bins with more protected entities. Setting `suppression threshold` to a higher value
+can lead to more suppression, but also to less distortion.
+
+### Other anonymization parameters
+
+There are in total eight parameters and affect the strength of anonymization. They are
```
Comment (edongashi) [inline docs/operation.md:142]: **and** affect the strength of anonymization? Maybe `that affect`

**Review by edongashi — DISMISSED** [review]

**Comment by yoid2000** [comment]
Edon I made the changes you suggested. Ready for merge.

**Review by edongashi — APPROVED** [review]

**Pull request merged**